src/index.html

```diff
@@ -7,6 +7,54 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="
+        calendar__day
+        calendar__day--start-day"
+        >
+        <p class="
+          calendar__num
+          calendar__num--day--1"
+        ></p>
+      </div>
+      <div class="
+        calendar__day
+        calendar__day--start-day--tue"
+        >
+        <p class="
+          calendar__num
+          calendar__num--day--2"
+        ></p>
+      </div>
```

Comment:
it must be just 
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

Reply:
Still not fixed from the previous review
<img width="891" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/64750958/f98ce199-82ec-4909-a2cb-f23956a8e37e">